src/constants.js

```diff
@@ -18,6 +18,16 @@ export default {
   OPT_OUT: 'amplitude_optOut',
   USER_ID: 'amplitude_userId',
 
+  // indexes of properties in storage format
+  DEVICE_ID_INDEX: 0,
```

Comment:
really appreciate that these are being constantized. One thought though - we should maybe note that these are part of essentially cookies v2 maybe? or just some versioning on cookie logi